pages/content/amp-dev/documentation/guides-and-tutorials/learn/experimental.md

```diff
@@ -64,3 +64,23 @@ Document can choose to opt in a certain experiments. To do that, simply put a me
 ```
 
 By doing so, the specified experiments will be enabled for all visitors of the document. However, not all experiments allow document-level opt-in. For a full list of whitelisted experiments, see the `allow-doc-opt-in` attribute in the project's [`prod-config.json`](https://github.com/ampproject/amphtml/blob/master/build-system/global-configs/prod-config.json) file. Note that document opt-in can be overridden by user opt-out.
+
+## Enable an Origin Trial
```

Comment:
IMO this documentation isn't actionable since origin trials are ephemeral and only apply to specific experiments, so I'd keep it to just a short note that they exist... sometimes.

Reply:
Crystal made some edits here does this look better to you @choumx?